lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -339,15 +344,26 @@ public static String getMavenPropertyInRequest(IPositionRequest request) {
 		int indexCloseBefore = beforeHoverText.lastIndexOf('}');
 		int indexCloseAfter = afterHoverText.indexOf('}');
 		if (indexOpen > indexCloseBefore) {
-			return tagText.substring(indexOpen + 2, indexCloseAfter + beforeHover);
+			Position propertyStart;
+			try {
+				String propertyText = tagText.substring(indexOpen + 2, indexCloseAfter + beforeHover);
+				propertyStart = new Position(request.getPosition().getLine(),
+						request.getXMLDocument().positionAt(request.getNode().getStart()).getCharacter() + indexOpen);
+				
+				// 3 is added to the end position's character to compensate for the '${}' 
+				Range propertyRange = new Range(propertyStart, new Position(request.getPosition().getLine(),
```

Comment:
This works although it could probably be a bit more elegant, but shouldn't be a blocker

Reply:
similarly
```java
int textStart = request.getNode().getStart();
Range propertyRange = XMLPositionUtility.createRange(textStart + indexOpen + 2, textStart + indexCloseAfter - 1, request.getXMLDocument());
```